web/src/settings_components.ts

```diff
@@ -893,6 +894,79 @@ export function check_property_changed(
     return current_val !== proposed_val;
 }
 
+export function populate_data_for_request(
+    subsection: JQuery,
```

Comment:
This should probably get renamed to `$subsection_elem` in a follow-up commit, since it's our pattern to decorate these names with the `$` marker.

Reply:
#30145